packages/terra-list/src/List.jsx

```diff
@@ -98,9 +109,21 @@ const List = ({
   refCallback,
   role,
   ariaSelectionStyle,
+  isDraggable,
+  onDragEnd,
   ...customProps
 }) => {
-  const theme = React.useContext(ThemeContext);
+  const theme = useContext(ThemeContext);
+  const [listItem, setlistItem] = useState([]);
+
+  useEffect(() => {
+    if (!(Array.isArray(children))) {
+      if (children) setlistItem([children]);
```

Comment:
```suggestion
      setlistItem([children]);
```
isArray will return false for undefined values as well